README.md

```diff
@@ -79,15 +79,19 @@ make testacc TESTARGS='-run=TestAccXXX'
 
 To contribute, please read the contribution guidelines: [Contributing to Terraform - Anexia Cloud Provider](CONTRIBUTING.md)
 
+1. Make your code changes on the branch of your fork of [terraform-provider-anxcloud](https://github.com/anexia-it/terraform-provider-anxcloud)
+2. Create a pull request
+3. Add your changes to the **Unreleased** section in [CHANGEME.md](CHANGEME.md)
+4. Trigger the integration tests via a `/ok-to-test sha=$SHA` comment. The $SHA represents the last commit in the PR.
+5. Merge the PR into the main branch of [terraform-provider-anxcloud](https://github.com/anexia-it/terraform-provider-anxcloud)
```

Comment:
Another thing not done by outside contributors and to clarify - just a "we will issue ok-to test" and "we will review, discuss and finally merge" kinda thing